### PR TITLE
Tab UI tutorial

### PR DESCRIPTION
**Pull request by Nibba2018**
Tutorial for Tab UI. Kindly merge #252 before merging this PR.

### CODE REVIEW

File: docs/tutorials/02_ui/viz_tab.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+import numpy as np
+
+###############################################################################
+# First, we create the Tab UI.
+
+tab_ui = ui.TabUI(position=(49, 94), size=(300, 300), nb_tabs=3,
+                  draggable=True)
+
+###############################################################################
+# Slider Controls for a Cube
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_tab.py:25]: add information about the tab number even if it is present below.

File: docs/tutorials/02_ui/viz_tab.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+###############################################################################
+# After defining content, we define properties for the tab.
+
+tab_ui.tabs[0].title = "Sliders"
+tab_ui.tabs[0].add_element(ring_slider, (0.3, 0.3))
+tab_ui.tabs[0].add_element(line_slider_x, (0.0, 0.0))
+tab_ui.tabs[0].add_element(line_slider_y, (0.0, 0.1))
+
+###############################################################################
+# CheckBoxes For Cylinder and Sphere
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_tab.py:83]: add information about the tab number

File: docs/tutorials/02_ui/viz_tab.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+checkbox.on_change = set_figure_visiblity
+
+###############################################################################
+# After defining content, we define properties for the tab.
+
+tab_ui.tabs[1].title = "Checkbox"
+tab_ui.tabs[1].add_element(checkbox, (0.2, 0.2))
+
+###############################################################################
+# Color Combobox for Fury
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_tab.py:126]: add information about the tab number

File: docs/tutorials/02_ui/viz_tab.py [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+
+
+###############################################################################
+# Next we prepare the scene and render it with the help of show manager.
+
+sm = window.ShowManager(size=(800, 500), title="Viz Tab")
+sm.scene.add(tab_ui, cube, cylinder, sphere, label)
+
+# To interact with the ui set interactive = True
+interactive = True
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_tab.py:207]: `False` instead of `True`

**Comment by Nibba2018** [comment]
Hello @skoudoro , I have made the necessary changes.

File: docs/tutorials/02_ui/viz_tab.py [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+                                text_alignment="Top")
+
+line_slider_y = ui.LineSlider2D(initial_value=0,
+                                min_value=-10, max_value=10,
+                                orientation="vertical",
+                                text_alignment="Right")
+
+cube = actor.box(centers=np.array([[10, 0, 0]]),
+                 directions=np.array([[0, 1, 0]]),
+                 colors=np.array([[0, 0, 255]]),
```
Comment (skoudoro) [inline docs/tutorials/02_ui/viz_tab.py:45]: Can you rebase and use `1`instead of `255`

Comment (Nibba2018) [reply docs/tutorials/02_ui/viz_tab.py]: @skoudoro Updated 😄

**Comment by skoudoro** [comment]
OK, it is ready to go, nice tutorial! merging

**Pull request merged**